cmd/repo-init/main.go

```diff
@@ -636,6 +636,8 @@ func determineWorkflowFromClusterPorfile(clusterProfile api.ClusterProfile) *str
 	switch clusterProfile {
 	case api.ClusterProfileAWS:
 		ret = "ipi-aws"
+	case api.ClusterProfileAWSArm64:
+		ret = "ipi-aws"
```

Comment:
note that I'm planning to adapt the `ipi-aws` workflow to work with arm64. see https://github.com/openshift/release/pull/19631